### DISCUSSION TITLE
Overture Maps Foundation - big opportunity for Organic Maps

### DISCUSSION THREAD

**Discussion by dumblob** [organicmaps/organicmaps#4102] [Ideas]
Linux Foundation established a a joint new non-profit organization *Overture Maps Foundation*.

https://www.linuxfoundation.org/press/linux-foundation-announces-overture-maps-foundation-to-build-interoperable-open-map-data

Main stakeholders seem to be Amazon, Meta, Microsoft, TomTom so far.

Organic Maps could benefit from this in number of ways. Feel free to link and mention your ideas below.

**Comment by dumblob** [comment]
### Traffic Data

Could be provided by Overture Maps Foundation infrastructure if any.

Related:
https://github.com/organicmaps/organicmaps/issues/1160
https://github.com/organicmaps/organicmaps/issues/588

**Reply by rtsisyk** [reply]
Already emailed them!

**Comment by biodranik** [comment]
Big tech here tries to save money by sharing internal datasets/OSM data fixes with each other. That is the main goal.

**Comment by nep2ner** [comment]
https://overturemaps.org/overture-maps-foundation-releases-first-world-wide-open-map-dataset/

**Comment by ooosssay** [comment]
POI data but

- Have a confidence of over 0.90 (or _maybe_ 0.85)
- If POI of same name already exists in OSM then Overture data should be added on top of it (Similar also with TIGER addresses not being shown when already exists in OSM)
- Overture POIs should be always hidden when above certain features where it isn’t possible for it to exist (ex. highways) unless the user strictly searches it. (This only applies to Overture data not OSM)

POI’s have addresses attached to them but its in freeform.

**Reply by map-per** [reply]
The quality of the Overture Place data is so low that I don't think it is worth all the filtering and duplicate elimination effort. Plus it will cause problems with Organic Maps build in OSM editor. 

It looks like e.g. Microsoft doesn't share the data they use on Bing Maps but something else.

**Reply by ooosssay** [reply]
I agree that Overture data is extremely low especially when it’s compared to areas in OSM (ex. Parts of Europe, Japan, popular cities) I think that (at least) Overture could be used in countries where they don’t have the resources to contribute to OSM (ex. Mexico, Peru, Singapore), this problem unfortunately will force people of these countries to have to stick to other Map Apps. It will take a long time for these countries to get to the same contribution activity as a place like, for example, Paris. Also possibly make the data as a separate download/opt-in or out.

Here’s an image comparison in OSM and Overture (screenshot is in Mexico):
![IMG_8405](https://github.com/user-attachments/assets/111246a0-db4f-4af3-a099-5b909b3cb171)
![IMG_8404](https://github.com/user-attachments/assets/c46ee3fa-198c-4e32-923d-9371ebe02fc1)



Additionally, tackling the problem about the built in OSM editor, there should be a questionnaire about if the poi exists and any other reasonable questions, this will then be sent as a note to OSM for someone to verify (since Overture is compatible with OSM). But then there’s the problem of removing the poi. But that gave me an idea! If Organic Maps (and other apps) would use Overture data, why not if the user says the place doesn’t exist it should be sent to Overture in which then the poi’s confidence level should be brought way down since this is essentially human review. (Essentially a collaboration between Map apps and Overture.)

Also most of the poi data comes from (from what I’ve seen) Meta and OSM.
